### PR TITLE
Add species master

### PR DESCRIPTION
**Pull request by captainrandom**
#56
#36

addresses #56 and #36

Unit tests pass. I copied this from one of my other branches.

### CODE REVIEW

**Comment by captainrandom** [comment]
This is what I get from running:

```curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$limit=50000' \
          | node parse-trees.js```

<img width="1792" alt="Screen Shot 2020-01-13 at 8 09 50 PM" src="https://user-images.githubusercontent.com/13500774/72313711-0f8e9c80-3641-11ea-89b7-43da073998ff.png">

**Comment by captainrandom** [comment]
This is me running the find missing species script
Screen Shot 2020-01-13 at 8 15 34 PM

**Comment by captainrandom** [comment]
I also added a script for local python setup.

**Comment by emillipede** [comment]
see #65

**Comment by captainrandom** [comment]
Screen Shot 2020-01-13 at 8 34 35 PM

File: find_missing_species.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    # verify that both dataframes have the join column
+    if species_id_col_name not in species_df.columns:
+        raise ValueError(f'species_df does not have column: {species_id_col_name}')
+    elif species_id_col_name not in trees_inventory_df.columns:
+        raise ValueError(f'trees_inventory_df does not have column: {species_id_col_name}')
+
+    # do a left join to find rows with new species ids and print them out
+    new_species_tree_inventory_df = filter_new_species_ids(trees_inventory_df, species_df)
+    print(new_species_tree_inventory_df.to_csv(index=False))
```
Comment (emillipede) [inline find_missing_species.py]: please add line to add/update the "missing species" trees to a csv in the repository (maybe in the data folder?)

File: setup_python_locally.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```
Comment (emillipede) [inline setup_python_locally.sh]: please put this part into a separate PR

File: find_missing_species.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    parser.add_argument('-s', '--species-attributes-csv',
+        default='data/species_attributes.csv',
+        help='file path for species_attributes.csv')
+
+    return parser.parse_args()
+
+
+def filter_new_species_ids(trees_inventory_df, species_df):
+    """
+    This finds full rows (aka all columns) from trees_inventory_df where
```
Comment (emillipede) [inline find_missing_species.py]: please document how to run this script (from command line?)

Comment (captainrandom) [reply find_missing_species.py]: Done. See readme.

File: setup_python_locally.sh (hunk 2 above)
Comment (captainrandom) [reply setup_python_locally.sh]: Done: https://github.com/Public-Tree-Map/public-tree-map-data-pipeline/pull/83

File: find_missing_species.py (hunk 1 above)
Comment (captainrandom) [reply find_missing_species.py]: Done (check latest version)
